storage/volumes.md

```diff
@@ -512,12 +517,64 @@ $ docker volume create \
 	--name cif-volume
 ```
 
-Notice the `addr` option is required if using a hostname instead of an IP so docker can perform the hostname lookup.
+The `addr` option is required if you specify a hostname instead of an IP.
+This lets Docker perform the hostname lookup.
+
+### Mount a block storage device
+
+You can mount a block storage device, such as an external drive or a drive partition, to a container.
+The following example shows how to create and use a file as a block storage device,
+and how to mount the file as a container volume.
+
+> **Note**
+>
+> The following procedure is only an example. This example creates an `ext4`
+> filesystem and mounted into a container. The filesystem support of your
+> system depends on the version of the Linux kernel you are using.
+>
+> Under the hood, the `--mount` flag invokes the Linux `mount` command and forwards
+> the options you pass to it unaltered. Docker doesn't implement any additional
+> functionality on top of the native mount feature supported by the Linux kernel.
```

Comment:
I think we could use the `mount` command call to illustrate what happens:

```bash
mount -t <mount.volume-opt.type> <mount.volume-opt.device> <mount.dst> -o <mount.volume-opts.o>
```

so the example
```bash
--mount='type=volume,dst=/external-drive,volume-driver=local,volume-opt=device=/dev/loop5,volume-opt=type=ext4'
```

is basically an equivalent to:
```bash
mount -t ext4 /dev/loop5 /external-drive
```